### PR TITLE
Changes to ref p38 dodal

### PR DESCRIPTION
**Pull request by keithralphs**

### CODE REVIEW

File: helm/blueapi/templates/deployment.yaml [hunk 1]
```diff
@@ -35,6 +35,9 @@ spec:
           - secret:
               name: {{ . }}
           {{- end }}
+      - name: plans
+        hostPath:
+          path: "/dls_sw/p38/software/bluesky_plans"
```
Comment (DiamondJoseph) [inline helm/blueapi/templates/deployment.yaml:40]: Un-hardcode this and get (this/these) from values.yaml if it exists

File: helm/blueapi/templates/deployment.yaml [hunk 2]
```diff
@@ -47,6 +50,12 @@ spec:
             - name: worker-config
               mountPath: "/config"
               readOnly: true
+            - name: plans
+              mountPath: "/plans"
+              readOnly: true
+          env:
+            - name: PYTHONPATH
+              value: /plans
```
Comment (DiamondJoseph) [inline helm/blueapi/templates/deployment.yaml:58]: Calllum wants to only allow python modules and not just a random folder

**Review by DiamondJoseph — CHANGES_REQUESTED** [review]

**Comment by stan-dot** [comment]
unclear ref changes, petition to close @callumforrester

**Comment by callumforrester** [comment]
Yep, no longer needed

**Pull request closed without merging**